src/worker-post.js

```diff
@@ -29,13 +29,16 @@
 		// #ifdef node
 		var data = event;
 		// #endif
+		const wasmMemory = new WebAssembly.Memory({ initial: data.initialMemory ?? 256, maximum: data.maxMemory ?? 256 });
```

Comment:
Since we don't have Babel or other transpiling going on here, using `??` would raise the Node.js version requirement from current 12 to 14, where nullish coalescing was added. I don't have super strong desires for bending over backwards to support old Node.js versions, but I think this one is easy enough to avoid. `||` should work fine here since `0` would be quite nonsensical value for these anyway.

```suggestion
		const wasmMemory = new WebAssembly.Memory({ initial: data.initialMemory || 256, maximum: data.maxMemory || 256 });
```